src/views/user/Profile.js

```diff
@@ -1,7 +1,111 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { userUid } from 'src/utils/userUid'
+import { axiosPrivate } from 'src/utils/axiosPrivate'
+import { CCard, CAvatar, CRow, CCol, CCardBody } from '@coreui/react'
+import avatarImg from '../../assets/images/avatars/6.jpg'
 
 const Profile = () => {
-  return <></>
+  const [userDetails, setUserDetails] = useState({ email: '', first_name: '', last_name: '' })
+  // let initials
+  // get user by user_uid
+  const getUser = async () => {
+    const res = await axiosPrivate.get(`/v1/user/${userUid()}/`)
+    setUserDetails(res.data)
+    console.log(res.data)
+  }
+  // initials = userDetails.first_name.substring(0,1).toUpperCase() + userDetails.last_name.substring(0,1).toUpperCase()
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  // format date function
+  const formattedDate = new Date(userDetails.date_joined).toDateString()
```

Comment:
In a subsequent PR we shall have one constant date formatter that will be called across the repo. For now, this will remain as is.

Reply:
I was to create a utility function that formats all dates. I understand we get both the date_joined and date of birth.